src/misc/00_intro.rst

```diff
@@ -6,6 +6,9 @@ Glossary
 
 .. glossary::
 
+  Recipe
+    **R**\ ecipe is the collection of files required to build the conda package. This includes at minimum the `meta.yaml <https://conda-forge.org/docs/maintainer/adding_pkgs.html#the-recipe-meta-yaml>`_, but can also include license files, patches, build scripts, test scripts etc.
```

Comment:
Recipe is the collection of files required to build a (not the) conda package. The includes (comma) at minimum (comma) the meta.yaml file...
Maybe you can add a link to the sample recipe. (https://github.com/conda-forge/staged-recipes/tree/master/recipes/example)

Reply:
Done, thanks for suggesting